scripts/pre_commit_hook.py

```diff
@@ -191,7 +192,7 @@ def check_changes_in_config() -> None:
 
 def run_prettier() -> None:
     """Runs prettier formatter."""
```

Comment:
@jayam04 Can you remove shell=True here. I think that's causing the problen why prettier isn't running. I found this 'When you add shell=True, and pass command line arguments as a list, only first element of the list will be executed. Rest will be ignored or would change sh behavior if it contains valid shell arguments". I cant really test this change because I took out the docker setup.